compiler/src/test/scala/org/scalajs/nscplugin/test/CallSiteInlineTest.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.nscplugin.test
+
+import util._
+
+import org.junit.Test
+import org.junit.Assert._
+
+import org.scalajs.ir.{Trees => js, Types => jstpe}
+import org.scalajs.ir.Names._
+
+class CallSiteInlineTest extends JSASTTest {
+  @Test
+  def testInline: Unit = {
+    val flags = {
+      """
+      object A {
+        println("F"): @inline
+      }
+      """.extractOne("println call") {
+        case js.Apply(flags, _, js.MethodIdent(name), _) if name.simpleName.nameString == "println" =>
```

Comment:
We should arguably construct the simple name beforehand here. However, I ran into some module initialization issues inside the `Name` object. Rings a bell?

Reply:
Hum. That doesn't ring a bell, no. :s